deploy/deploy.sh

```
@@ -0,0 +1,293 @@
+# ========================================
+# 함수 정의
+# ========================================
+
+# 환경변수 로딩
+load_env_file() {
+  ENV_FILE="$(dirname "$0")/.env"
+
+  if [ -f "$ENV_FILE" ]; then
+    log_info ".env 파일을 로드합니다: $ENV_FILE"
+    set -a
+    source "$ENV_FILE"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)